sdk/python/kfp_tekton/compiler/_op_to_template.py

```diff
@@ -133,20 +133,44 @@ def _op_to_template(op: BaseOp):
     outputs_dict = _outputs_to_json(op, processed_op.outputs, param_outputs, output_artifacts)
     if outputs_dict:
         template['spec']['results'] = []
+        outfile_step = {'command': ['/bin/bash'], 'image': 'ubuntu', 'name': 'outfile-step', 'args': ['-c', 'echo Copying files to Tekton dir...;']}
```

Comment:
I would not use ubuntu to copy files, that's a rather large image to just do a `cp`.

```
       - name: copy-results
          image: busybox
          script: |
            #!/bin/sh
            set -exo pipefail
            cp -r [something] [somewhere]
```